meerk40t/core/elements.py

```diff
@@ -470,10 +473,10 @@ def length(self, v):
         return float(Length(v))
 
     def length_x(self, v):
-        return float(Length(v, relative_length=self.device.width))
+        return float(Length(v, relative_length=self.device.width, unitless=1))
```

Comment:
What's the justification for this? I daresay there's a lot of cases when this function would absolutely expect that unitless values are pixels, since that's fairly consistent.

Reply:
Confirmed this would require that all things that use these functions will have a unitless value of 1 native scene unit. That's rect, line, circle, etc.